### PR TITLE
Consolidate xssh signature formats per Issue #217.

### PR DESCRIPTION
**Pull request by aaspring**
Output format --

```json
"server_signature": {
          "parsed": {
            "algorithm": "ssh-rsa",
            "value": "RT0dGAYUdJB2O/R4zT6f89M5s2kdPqKdjWLjlLQtpB2/RJO3wXkbF5KRUQq4+zXFPh8/cCLSLkQblZvScJ3pghi6nhMnYhQLL78HdPfI+KuDIfCptKwNsoysP36ya/jg8fFFWelO4B4RjUFlyZFNQhH5VNyI5tDI/mKJPn9hVps8Y2C55ctgiRkcAF+g7mO1+wQj4FYpE70sPJTz1gtKFMSa/JC4OXDwWzZo7cIwPus8GsGNpq3HqVcgKqnRcO8TgrvvLj/x0ec3I37rMU4XQDvk9fXigSbbpUgjKI592kmyO1gX1nO7sjEhks6O/PdZzYXbCMEbSCLs0+ouqGH+fQ=="
          },
          "raw": "AAAAB3NzaC1yc2EAAAEART0dGAYUdJB2O/R4zT6f89M5s2kdPqKdjWLjlLQtpB2/RJO3wXkbF5KRUQq4+zXFPh8/cCLSLkQblZvScJ3pghi6nhMnYhQLL78HdPfI+KuDIfCptKwNsoysP36ya/jg8fFFWelO4B4RjUFlyZFNQhH5VNyI5tDI/mKJPn9hVps8Y2C55ctgiRkcAF+g7mO1+wQj4FYpE70sPJTz1gtKFMSa/JC4OXDwWzZo7cIwPus8GsGNpq3HqVcgKqnRcO8TgrvvLj/x0ec3I37rMU4XQDvk9fXigSbbpUgjKI592kmyO1gX1nO7sjEhks6O/PdZzYXbCMEbSCLs0+ouqGH+fQ=="
        },
```

### CODE REVIEW

File: zgrab_schema.py [hunk 1]
```diff
@@ -743,6 +743,14 @@
     "public_bytes":Binary(),
 })
 
+xssh_signature = SubRecord({
+    "parsed":SubRecord({
+        "algorithm":String(),
+        "value":Binary(),
+    }),
+    "raw":Binary(),
```
Comment (dadrian) [inline zgrab_schema.py]: Is `raw` used anywhere? How is `raw` different from `value`?

**Comment by dadrian** [comment]
Also, it looks like tests aren't passing. The XSSH unmarshal tests don't seem to work.

**Comment by dadrian** [comment]
@aaspring bump

**Comment by aaspring** [comment]
I'm aware. I'm trying to knock out a bunch of ztls PRs that I need for a project. I will work on this as I'm waiting for those PRs to go through.

File: zgrab_schema.py (hunk 1 above)
Comment (aaspring) [inline zgrab_schema.py]: Yes -- raw is used to store the entire signature structure in certs and for server dh kex messages.

```raw``` is the entire signature structure including the algorithm, value, lengths, and any trailing data.

**Pull request merged**